packages/worker/src/api/routes/index.ts

```diff
@@ -33,10 +30,11 @@ export const routes: Router[] = [
   statusRoutes,
   selfRoutes,
   licenseRoutes,
-  userGroupRoutes,
-  auditLogRoutes,
+  pro.groups,
+  pro.auditLogs,
   migrationRoutes,
   accountRoutes,
   restoreRoutes,
   eventRoutes,
+  ...pro.scimRoutes,
```

Comment:
preferred if the exported prop is `scim` instead of `scimRoutes` to line up with the rest of the pro API exports.

Reply:
Should the other ones not be `groupsRoutes`, to be consistent with the worker?